implementations/rust/ockam/ockam_node/src/storage/database/migrations/20240108100000_rename_confluent_config_table.sql

```diff
@@ -0,0 +1,2 @@
+-- Rename the `confluent_config` table to `kafka_config`
+alter table confluent_config rename to kafka_config;
```

Comment:
As a matter of style I have previously tried to write all the SQL keywords in capital letters, `ALTER`, `TABLE`, etc... Maybe this is a useful convention to keep for readability.

Reply:
I usually try to not use that convention (IDE's syntax highlighting is good enough for me to keep the readability). All change it anyways to keep it consistent.